mijia-homie/mijia-homie.example.toml

```diff
@@ -1,13 +1,26 @@
 [homie]
+# The ID to use for the Homie device. This must be unique for a given prefix and server.
 device_id="mijia-bridge"
+# The human-readable name to use for the Homie device.
 device_name="Mijia bridge"
+# The Homie base MQTT topic.
 prefix="homie"
+# The name of the file containing sensor MAC address to name mappings.
 sensor_names_filename="sensor-names.toml"
+# The minimum time to wait between sending consecutive readings for the same sensor.
+min_update_period_seconds=0
```

Comment:
Worth noting here that the sensor is known to give updates every 6 seconds, so in practice you should pick a number slightly below the multiple-of-6 that you want, for consistent results.